csrc/device_lower/pass/inline_ptx.cpp

```diff
@@ -55,11 +57,13 @@ class LowerToInlinePtx : public kir::ExprMutator {
 
   void handle(LoadStoreOp* ldst) override {
     if (ir_utils::isLdMatrixOp(ldst)) {
-      auto op = ldst->opType();
       std::stringstream ss;
       ss << "ldmatrix.sync.aligned.x"
          << std::get<ArrayType>(ldst->out()->dtype().type).size;
-      if (op == LoadStoreOpType::LdMatrixTranspose) {
+      auto tv_consumer = ldst->out()->as<kir::TensorIndex>()->view();
+      auto tv_producer = ldst->in()->as<kir::TensorIndex>()->view();
+      if (!mma_utils::isConsumerAllocationInnerIDProducerAllocationInnerID(
```

Comment:
Instead of having `mma_utils::isConsumerAllocationInnerIDProducerAllocationInnerID`, should we instead have a
```
ir_utils::isLdMatrixTranspose(const LoadStoreOp*)
```